cluster/k8s-multus-1.10.4/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubernetes 1.10.4 with multus network plugin in ephemeral containers
```

Comment:
Why do you add this provider, I can see you do not use it under CI

Reply:
I used it before we have the 1.11.1 multus provider want me to add it under the CI or remove it?

Reply:
I prefer to remove because we already have too many jobs, thx.

Reply:
No problem